acrobat/scripts/scripts.js

```diff
@@ -336,3 +336,40 @@ const { ietf } = getLocale(locales);
     }
   }, 1000);
 }());
+
+// Process placeholders in the document
+const processPlaceholders = (documentElement) => {
+  const placeholderRegex = /{{(.*?)}}/g;
```

Comment:
@joaquinrivero assuming this to be specific to commerce (DC), we may have to use a different pattern from Milo placeholder pattern to avoid Milo placeholders feature replacing ours ?